mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/PersonItem.kt

```diff
@@ -26,15 +34,16 @@ import androidx.compose.ui.unit.dp
 @Composable
 fun PersonItem(
     person: Person,
+    onShowProfileCLick: () -> Unit,
     modifier: Modifier = Modifier,
     trailingAction: @Composable () -> Unit = {}
 ) {
   ListItem(
-      modifier = modifier,
+      modifier = Modifier.background(Beige050).clickable { onShowProfileCLick() },
```

Comment:
I think this should be `modifier`, since it's the top-level composable within `PersonItem`.

```suggestion
      modifier = modifier.background(Beige050).clickable { onShowProfileCLick() },
```